frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx

```diff
@@ -451,22 +443,21 @@ export const ProvidedAPIsPage = (props: ProvidedAPIsPageProps) => {
     [providedAPIs],
   );
 
-  const isNamespaced = providedAPIs.every((api) => {
+  const hasNamespacedAPI = providedAPIs.some((api) => {
```

Comment:
@TheRealJon The use of  `some` function wouldn't fix the other bug - [Namespace value is missing on the list when selecting "All namespaces" for operators](https://bugzilla.redhat.com/show_bug.cgi?id=2091545). I'm not sure how would want to handle this. Can we use hyphen `-` for missing namespace?